db/schema.rb

```diff
@@ -74,6 +74,31 @@
     t.index ["parent_id"], name: "fk_rails_4404228d2f"
   end
 
+  create_table "sign_up_teams", charset: "utf8mb4", collation: "utf8mb4_0900_ai_ci", force: :cascade do |t|
+    t.bigint "sign_up_topic_id", null: false
+    t.boolean "is_waitlisted"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.bigint "teams_id", null: false
+    t.index ["sign_up_topic_id"], name: "index_sign_up_teams_on_sign_up_topic_id"
+    t.index ["teams_id"], name: "index_sign_up_teams_on_teams_id"
+  end
```

Comment:
make sure not to leave out columns to ensure no data is lost in migration. 
preference_priority_number is missing